packages/inter-protocol/src/price/fluxAggregatorKit.js

```diff
@@ -109,64 +121,64 @@ export const makeFluxAggregator = async (
     unitAmountIn,
   });
 
+  const makeRoundsManagerKit = prepareRoundsManagerKit(baggage);
+  const makeOracleAdminKit = prepareOracleAdminKit(baggage);
+
   const makeRecorderKit = defineRecorderKit({
+    // @ts-expect-error XXX
     makeDurablePublishKit,
     makeRecorder,
   });
 
-  // For publishing priceAuthority values to off-chain storage
-  const { recorder: priceRecorder, subscriber: quoteSubscriber } =
-    makeRecorderKit(
-      storageNode,
-      /** @type {import('@agoric/zoe/src/contractSupport/recorder.js').TypedMatcher<PriceDescription>} */ (
-        M.any()
-      ),
-    );
+  // End makers
```

Comment:
stray marker, please drop.

Reply:
This is meaningful. It marks the end of the kind creations, such that awaits following it won't break reincarnation. 
https://github.com/Agoric/agoric-sdk/blob/4276860bb1904cdaae744b7676807c2e59574b88/packages/vat-data/README.md?plain=1#L23

Is there a clearer comment you would suggest? I'm inclined to use a foldable region but I recall some push-back on those

Reply:
Sorry, it looked like a stray endregion marker. It looked stray because I didn't see the matching `start`.  I like delimiting regions of code like this
```
// end of maker definitions /////////////////////////////////
```
I'm one of the people who push back on region markers.